<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteStudentDialog</class>
 <widget class="QDialog" name="DeleteStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* === –û—Å–Ω–æ–≤–Ω–æ–µ –æ–∫–Ω–æ –¥–∏–∞–ª–æ–≥–∞ === */


/* === QLineEdit –ø–æ–ª—è === */
QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 6px;
    padding: 6px 8px;
    selection-background-color: #a8d0ff;
}

/* === QTextEdit === */
QTextEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 6px;
    padding: 6px;
    selection-background-color: #a8d0ff;
}

/* === QPushButton === */
QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
}

QPushButton:hover {
    background-color: #357abd;
}

QPushButton:pressed {
    background-color: #2a5d9f;
}

/* === QDialogButtonBox –∫–Ω–æ–ø–∫–∏ === */
QDialogButtonBox QPushButton {
    min-width: 80px;
}

/* === QLabels === */
QLabel {
    font-weight: 500;
}

/* === QLineEdit –ø–æ–ª—è (—Ç—ë–º–Ω–∞—è —Ç–µ–º–∞) === */
QLineEdit {
    background-color: #2b2b2b;    /* —Ç—ë–º–Ω—ã–π —Ñ–æ–Ω */
    color: #ffffff;               /* –±–µ–ª—ã–π —Ç–µ–∫—Å—Ç */
    border: 1px solid #555555;    /* —Ç—ë–º–Ω–∞—è –≥—Ä–∞–Ω–∏—Ü–∞ */
    border-radius: 6px;
    padding: 6px 8px;
    selection-background-color: #4a90e2; /* —Ü–≤–µ—Ç –≤—ã–¥–µ–ª–µ–Ω–∏—è */
    selection-color: #ffffff;             /* —Ü–≤–µ—Ç —Ç–µ–∫—Å—Ç–∞ –ø—Ä–∏ –≤—ã–¥–µ–ª–µ–Ω–∏–∏ */
}

/* === –≠—Ñ—Ñ–µ–∫—Ç –ø—Ä–∏ —Ñ–æ–∫—É—Å–µ === */
QLineEdit:focus {
    border: 2px solid #4a90e2;
    outline: none;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>541</width>
     <height>167</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditFile">
      <property name="placeholderText">
       <string>–§–∞–π–ª —Å–æ —Å—Ç—É–¥–µ–Ω—Ç–∞–º–∏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditName">
      <property name="placeholderText">
       <string>–§–ò–û</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditBall">
      <property name="placeholderText">
       <string>–ë–∞–ª–ª(–¥–ª—è —Ç–æ—á–Ω–æ–≥–æ —É–¥–∞–ª–µ–Ω–∏—è —Ä–µ–∫–æ–º–µ–Ω–¥—É–µ—Ç—å—Å—è) –Ω–µ –æ–±—è–∑—è—Ç–µ–ª—å–Ω–æ–µ –ø–æ–ª–µ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBrowse">
      <property name="text">
       <string>üìÇ–í—ã–±—Ä–∞—Ç—å</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
